lighthouse-core/audits/audit.js

```diff
@@ -57,6 +57,60 @@ class Audit {
   }
 
   /**
+   * @param  {!Audit.Headings} headings
```

Comment:
just one space (here and elsewhere). Is this a vscode setting that needs to be changed?